### PR TITLE
Remove duplicate docs for SHOW INDEX, SHOW KEYS infavor of SHOW INDEXES

### PR DESCRIPTION
**Pull request by enkilee**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
fix #https://github.com/pingcap/docs/pull/15364

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15364
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by ran-huang** [comment]
Please note that PR titles should briefly summarize the content of your changes. For details, see [PR Checklist](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#pr-checklist).

File: sql-statements/sql-statement-show-indexes.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: SHOW INDEXES [FROM|IN]
 summary: TiDB 数据库中 SHOW INDEXES [FROM|IN] 的使用概况。
-aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-indexes/','/docs-cn/dev/reference/sql/statements/show-indexes/']
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-indexes/','/docs-cn/dev/reference/sql/statements/show-indexes/','/docs-cn/dev/sql-statement-show-index/','/docs-cn/dev/sql-statement-show-keys/']
```
Comment (ran-huang) [inline sql-statements/sql-statement-show-indexes.md:4]:
```suggestion
aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-indexes/','/docs-cn/dev/reference/sql/statements/show-indexes/','/zh/tidb/dev/sql-statement-show-index/','/zh/tidb/dev/sql-statement-show-keys']
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: sql-statements/sql-statement-show-indexes.md (hunk 1 above)
Comment (enkilee) [reply sql-statements/sql-statement-show-indexes.md]: Roger

File: sql-statements/sql-statement-show-indexes.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: SHOW INDEXES [FROM|IN]
 summary: TiDB 数据库中 SHOW INDEXES [FROM|IN] 的使用概况。
-aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-indexes/','/docs-cn/dev/reference/sql/statements/show-indexes/']
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-indexes/','/docs-cn/dev/reference/sql/statements/show-indexes/','/zh/tidb/dev/sql-statement-show-index/','//zh/tidb/dev/sql-statement-show-keys/']
```
Comment (ran-huang) [inline sql-statements/sql-statement-show-indexes.md:4]:
```suggestion
aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-indexes/','/docs-cn/dev/reference/sql/statements/show-indexes/','/zh/tidb/dev/sql-statement-show-index/','/zh/tidb/dev/sql-statement-show-keys/']
```

File: sql-statements/sql-statement-rename-index.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ Create Table: CREATE TABLE `t1` (
 ## 另请参阅
 
 * [SHOW CREATE TABLE](/sql-statements/sql-statement-show-create-table.md)
-* [CREATE INDEX](/sql-statements/sql-statement-create-index.md)
+* [CREATE INDEXS](/sql-statements/sql-statement-create-indexes.md)
```
Comment (ran-huang) [inline sql-statements/sql-statement-rename-index.md:86]:
```suggestion
* [CREATE INDEX](/sql-statements/sql-statement-create-index.md)
```

File: sql-statements/sql-statement-rename-index.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ Create Table: CREATE TABLE `t1` (
...
 * [DROP INDEX](/sql-statements/sql-statement-drop-index.md)
 * [SHOW INDEX](/sql-statements/sql-statement-show-index.md)
```
Comment (ran-huang) [inline sql-statements/sql-statement-rename-index.md:88]:
```suggestion
* [SHOW INDEXES](/sql-statements/sql-statement-show-indexes.md)
```

**Review by ran-huang — CHANGES_REQUESTED** [review]

File: sql-statements/sql-statement-rename-index.md (hunk 4 above)
Comment (enkilee) [reply sql-statements/sql-statement-rename-index.md:88]: sync change file sql-statement-show-index.md name to sql-statement-show-indexes.md ?

File: sql-statements/sql-statement-rename-index.md [hunk 5]
```diff
@@ -83,7 +83,7 @@ Create Table: CREATE TABLE `t1` (
 ## 另请参阅
 
 * [SHOW CREATE TABLE](/sql-statements/sql-statement-show-create-table.md)
-* [CREATE INDEX](/sql-statements/sql-statement-create-index.md)
+* [CREATE INDEXES](/sql-statements/sql-statement-create-indexes.md)
```
Comment (ran-huang) [inline sql-statements/sql-statement-rename-index.md:86]:
```suggestion
* [CREATE INDEX](/sql-statements/sql-statement-create-index.md)
```

File: sql-statements/sql-statement-rename-index.md
Comment (ran-huang) [inline sql-statements/sql-statement-rename-index.md:1]: Please check out the original PR (https://github.com/pingcap/docs/pull/15364/files#diff-c56aefa2408bea8ffa7f6003659c678750cf4d111292ab8a19241101392e80d0R62) and align with it.

Comment (enkilee) [reply sql-statements/sql-statement-rename-index.md:1]: OK

**Comment by ran-huang** [comment]
The rest LGTM, but the file [`sql-statements/sql-statement-rename-index.md`](https://github.com/pingcap/docs/pull/15364/files#diff-c56aefa2408bea8ffa7f6003659c678750cf4d111292ab8a19241101392e80d0R62) is not modified the same way as in the English PR.

**Comment by enkilee** [comment]
> The rest LGTM, but the file [`sql-statements/sql-statement-rename-index.md`](https://github.com/pingcap/docs/pull/15364/files#diff-c56aefa2408bea8ffa7f6003659c678750cf4d111292ab8a19241101392e80d0R62) is not modified the same way as in the English PR.

sorry. fix it.

**Review by ran-huang — APPROVED** [review]

**Pull request merged**